FirebaseFunctions/Sources/Functions.swift

```diff
@@ -38,6 +38,13 @@ enum FunctionsConstants {
   static let defaultRegion = "us-central1"
 }
 
+extension Functions {
+  enum Endpoint {
```

Comment:
Could this type be made private?

Reply:
I guess it can be removed altogether if `HTTPSCallable` is refactored to store the actual URL.